core/src/main/resources/reference.conf

```diff
@@ -34,6 +34,18 @@ akka.persistence.r2dbc {
       throughput-collect-interval = 10 seconds
     }
 
+    # Group the slices for an entity type into this number of topics. Most efficient is to use
+    # the same number as number of projection instances. If configured to less than the number of
+    # of projection instances the overhead is that events will be sent more than once and discarded
+    # on the destination side. If configured to more than the number of projection instances
+    # the events will only be sent once but there is a risk of exceeding the limits of number
+    # of topics that PubSub can handle (e.g. OversizedPayloadException).
+    # Must be between 1 and 1024 and a whole number divisor of 1024 (number of slices).
+    # This configuration can be changed in a rolling update, but there might be some events
+    # that are not delivered via the pub-sub path and instead delivered later by the queries.
+    # This configuration cannot be defined per journal, but is global for the ActorSystem.
+    publish-events-number-of-topics = 128
```

Comment:
with hindsight it would have been better to structure all `publish-events` properties in its own config section, but not worth breaking